app/ownedFile.js

```diff
@@ -8,7 +8,6 @@ export default class OwnedFile {
     this.url = obj.url;
     this.name = obj.name;
     this.size = obj.size;
-    this.type = obj.type;
```

Comment:
cryptically this was related to old metrics. `type` was 'click' or 'drag'. no longer used